integration/spark/src/main/common2.3and2.4/org/apache/spark/sql/SparkVersionAdapter.scala

```diff
@@ -430,6 +430,22 @@ trait SparkVersionAdapter {
   def evaluateWithPredicate(exp: Expression, schema: Seq[Attribute], row: InternalRow): Any = {
     InterpretedPredicate.create(exp, schema).expression.eval(row)
   }
+
+  def getUpdatedPlan(plan: LogicalPlan, sqlText: String): LogicalPlan = {
+    plan match {
+      case create@CreateTable(tableDesc, mode, query) =>
+        if ( tableDesc.storage.locationUri.isDefined &&
+             !sqlText.toUpperCase.startsWith("CREATE EXTERNAL TABLE ")) {
+          // add a property to differentiate if create table statement has external keyword or not
+          val newProperties = tableDesc.properties. +("hasexternalkeyword" -> "false")
+          val updatedTableDesc = tableDesc.copy(properties = newProperties)
+          CreateTable(updatedTableDesc, mode, query)
```

Comment:
can we make the table type as external , instead of putting in properties. 
This was no need to check for this keyword and the existing flows can handle the transactional table creation

Reply:
ok